routes/carts.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database/index');
+
+router.get('/:user_id', (req, res) => {
+  let userId = req.params.user_id;
+  return knex
+    .raw('SELECT * FROM users WHERE id = ?', [userId])
+    .then((user) => {
+      if (!user || !user.rowCount) {
+        throw `{USER NOT FOUND}`;
+      }
+      return knex.raw(
+        `SELECT products.* FROM carts JOIN products ON (carts.product_id = products.id) JOIN users ON (carts.user_id = users.id) WHERE user_id = ?`,
+        [user.rows[0].id],
+      );
+    })
+    .then((returnProducts) => {
+      res.send(returnProducts.rows);
+    })
+    .catch((err) => {
```

Comment:
This catch should be for any errors that aren't catched above and should return a 500 code.